.pre-commit-config.yaml

```diff
@@ -30,3 +30,7 @@ repos:
       - id: flake8
         args:
         - --show-source
+  - repo: https://github.com/sirosen/check-jsonschema
+    rev: 0.19.2
+    hooks:
+        - id: check-github-workflows

```

Comment:
I think we want check-jsonschema here, not check-github-workflows. And we may have to specify the schema file: https://check-jsonschema.readthedocs.io/en/latest/precommit_usage.html#check-jsonschema

Reply:
Yes, of course. I added this to the hooks and forgot to expand on that. Will fix! 👍🏻 That said, we can still *also* check the GitHub actions workflow definition files. 😁

Reply:
Agreed! Both would be good.

Reply:
Done.